github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -82,6 +87,42 @@ public boolean apply(GitHubPRCause cause) {
                 }
             }
 
+            if (trigger.isAbortRunning()) {
+                Computer[] computers = getJenkinsInstance().getComputers();
+                for (Computer computer : computers) {
+                    if (isNull(computer)) {
+                        continue;
+                    }
+
+                    List<Executor> executors = computer.getExecutors();
```

Comment:
Consider also `getOneOffExecutors`, at least if you care about Pipeline compatibility. (Not sure about `matrix-project`.)